Add GEFS atm-only fcst job support

WalterKolczynski-NOAA opened the pull request:
**Description**

Makes the minor changes necessary to run GEFS atm-only fcst job only from pre-staged ICs. Also many unnecessary DA settings are removed from the GEFS version of `config.base`. A few settings are retained either because they are needed, or are expected to be needed to support running from reanalysis (e.g. `IAU_FHROT`).

Resolves #826

**Type of change**

- [x] New feature (non-breaking change which adds functionality)

**How Has This Been Tested?**

- [x] 2+ctrl member GEFS forecast on Orion
  
**Checklist**

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] New and existing tests pass with my changes
- [x] Any dependent changes have been merged and published

WalterKolczynski-NOAA committed:
Add GEFS fcst job support

Makes the minor changes necessary to run GEFS fcst job only from pre-staged
ICs. Also many unnecessary DA settings are removed from the GEFS version of
`config.base`.

Resolves #826

aerorahul commented on jobs/JGLOBAL_FORECAST:
```diff
@@ -17,15 +17,23 @@ rCDUMP=${CDUMP}
 # Forecast length for GFS forecast
-if [ ${CDUMP} = "gfs" ]; then
+case ${CDUMP} in
+  gfs | gefs)
```
In the `gefs` tree, does `FHMAX_GFS` still make sense?
Should the line 23 be something like:
```
current_var=FHMAX_${CDUMP^^}; declare -x FHMAX=${current_var}
```
And so on.  
Beyond this point, there shouldn't be any references to `_gfs`, `_gefs` etc.

WalterKolczynski-NOAA replied:
Can, just makes the cases longer

WalterKolczynski-NOAA replied:
These kind of things are woven throughout the workflow, well beyond the little portion I am working on here.

aerorahul commented on parm/config/gefs/config.efcs:
```diff
@@ -66,32 +66,13 @@ export SPPT_LSCALE=500000.
-if [ $QUILTING = ".true." -a $OUTPUT_GRID = "gaussian_grid" ]; then
-    export DIAG_TABLE="$HOMEgfs/parm/parm_fv3diag/diag_table_da"
+if [[ ${QUILTING} = ".true." && ${OUTPUT_GRID} = "gaussian_grid" ]]; then
+    export DIAG_TABLE="${HOMEgfs}/parm/parm_fv3diag/diag_table_da"
 else
-    export DIAG_TABLE="$HOMEgfs/parm/parm_fv3diag/diag_table_da_orig"
+    export DIAG_TABLE="${HOMEgfs}/parm/parm_fv3diag/diag_table_da_orig"
```
Dial this down to one table.  There is no `_da` counterpart in GEFS

WalterKolczynski-NOAA replied:
Actually, we can just remove the whole thing. We don't need to change the diag table set in `config.fcst`.

WalterKolczynski-NOAA replied:
And done

aerorahul commented on jobs/JGLOBAL_FORECAST:
```diff
@@ -17,15 +17,23 @@ rCDUMP=${CDUMP}
-fi
+    ;;
+  *)
+    echo "FATAL ERROR: Unknown CDUMP '${CDUMP}'"
```
Should we use `RUN` instead of `CDUMP` here and in the case statement?

WalterKolczynski-NOAA replied:
Can, just makes the cases longer

WalterKolczynski-NOAA replied:
Done. Doesn't really make the case statements longer once I thought about it.

WalterKolczynski-NOAA committed:
Stop using DA diag table from GEFS, remove unneeded settings

The `config.efcs` was using the DA diag tables for GEFS when it should
just use the normal diag table from `config.fcst`.

Also removed more unneeded DA settings from the GEFS side.

Refs: #826

TerrenceMcGuinness-NOAA commented:
@aerorahul said he had a fix for the **vrfy** file move issue?

WalterKolczynski-NOAA commented:
> @aerorahul said he had a fix for the **vrfy** file move issue?

His solution was to put the commit hash as part of the pslot.

TerrenceMcGuinness-NOAA commented:
Oh, I see, didn't know those where related, makes total sense now, ok I"ll test that.

WalterKolczynski-NOAA commented:
> Oh, I see, didn't know those where related, ok I"ll test that.

Well, the path has the pslot in it, and if you use the commit hash that will always be unique unless you rerun a PR without any changes.

That said, given `gdasvrfy` is now dying the first time running and it isn't a clobber thing, I think something else is going on.

WalterKolczynski-NOAA committed:
Use RUN instead of CDUMP to determine output frequencies in forecast

Refs: #826

aerorahul reviewed (approved):
looks good.
